src/net.cpp

```diff
@@ -2149,9 +2150,9 @@ void CConnman::ThreadOpenMasternodeConnections()
                         }
                         const auto& addr2 = dmn->pdmnState->addr;
                         if (!connectedNodes.count(addr2) && !IsMasternodeOrDisconnectRequested(addr2) && !connectedProRegTxHashes.count(proRegTxHash)) {
-                            auto addrInfo = addrman.GetAddressInfo(addr2);
+                            int64_t lastAttempt = mmetaman.GetMetaInfo(dmn->proTxHash)->GetLastOutboundAttempt();
```

Comment:
`mmetaman.GetMetaInfo(dmn->proTxHash)`

Possible for this to return null? Same for below.

Reply:
Nope, `GetMetaInfo` will create the entry if not present yet.